README.md

```diff
@@ -268,4 +268,47 @@ From here, fill in the methods in AbcTask.cxx, and AbcCheck.cxx if needed.
 
 In case special additional dependencies are needed, create a new bucket in QualityControlModules/cmake/QualityControlModulesDependencies.cmake.
 
+## QC in Data Processing Layer framework
 
+Quality Control has been adapted to be used as Data Processor in
+[O2 framework](https://github.com/AliceO2Group/AliceO2/tree/dev/Framework/Core#data-processing-layer-in-o2-framework).
+Keep in mind, that checkers are not available at this moment.
+To add a QC task into workflow:
+
+1. Create your module using SkeletonDPL as a base. Refer to the steps mentioned
+in the chapter [Modules development](https://github.com/AliceO2Group/QualityControl#modules-development)
+- they are the same.
+2. Define inputs and parameters of your QC Task in .ini config file. Use
+[Framework/qcTaskDplConfig.ini](https://github.com/AliceO2Group/QualityControl/blob/master/Framework/qcTaskDplConfig.ini)
+as a reference.
```

Comment:
Add the name of the section and variables to update. What should I put for `outputDataOrigin` and `outputDataDescription` ? and what is `inputs` ?